system-addon/lib/TopSitesFeed.jsm

```diff
@@ -17,15 +16,18 @@ XPCOMUtils.defineLazyModuleGetter(this, "Screenshots",
 
 const TOP_SITES_SHOWMORE_LENGTH = 12;
 const UPDATE_TIME = 15 * 60 * 1000; // 15 minutes
+const DEFAULT_SITES_PREF = "default.sites";
```

Comment:
k88: Doesn't a `.`  usually denote a branch?
Maybe? I suppose we could have different defaults under that "branch" but one tricky aspect of changing this right now is that all the default test prefs need to be renamed in various files.